### ISSUE TITLE
Feature request: add ring to show touch area 

### ISSUE THREAD

**Issue by Akshar062** [MikeOrtiz/TouchImageView#617]
it would be nice to have ring to show where are we touching it will be very helpful when mirroring ring also get mirror and will change position when other image's ring has change

**Comment by hannesa2** [comment]
Yes this would be a nice option https://github.com/MikeOrtiz/TouchImageView?tab=readme-ov-file#project-status-maintenance-mode
